File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n');
const N = +input[0];
const members = input.slice(1).map(x => x.split(' '));
console.log(members.sort((a, b) => +a[0] - +b[0]).map(x => x.join(' ')).join('\n'));

File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.py
N = int(input())
members = []

for _ in range(N):
    age, name = input().split()
    members.append([int(age), name])
    
members.sort(key=lambda x: x[0])
print('\n'.join([f'{str(age)} {name}' for age, name in members]))